mandatory/2-function-creation.js

```diff
@@ -15,26 +23,56 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice,discount) {
```

Comment:
Issue: There's no need to take `discount` as a parameter to this function. You should only define the inputs that are needed by the function or expected by the tests.